docs/app/Examples/modules/Sticky/Types/StickyAdjacentContextExample.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react'
+import { Grid, Header, Image, Rail, Segment, Sticky } from 'semantic-ui-react'
+
+const Placeholder = () => (
+  <Image
+    style={{ marginTop: 14, marginBottom: 14 }}
+    className='wireframe paragraph image'
+    src='/assets/images/wireframe/paragraph.png'
```

Comment:
Please sort props in an alphabetical order, eg. `className`, `src`, `style`. `style` object members, too.